skbio/io/format/embl.py

```diff
@@ -1374,45 +1352,46 @@ def _serialize_sequence(obj, indent=5):
     SQ = SQ.format(size=len(obj), unit=obj.metadata["LOCUS"]["unit"].upper(),
                    n_a=n_a, n_c=n_c, n_g=n_g, n_t=n_t, n_others=n_others)
 
-    for i in range(0, len(seq), line_size):
-        line = seq[i:i+line_size]
-        s = '{key:<{indent}}{s}'.format(
-            key='', indent=indent, s=chunk_str(line, frag_size, ' '))
+    # fasta sequence will have indent spaces on the left, chunk_size/frag_size
+    # groups of frag_size letters separated by n-1 groups of single spaces,
+    # then the sequence length aligned on the right to get a string of
+    # line_size. Calculate padding for semplicity
+    n_frags = int(chunk_size / frag_size)
+    pad_right = chunk_size + n_frags-1
+    pad_left = line_size - pad_right - indent
```

Comment:
Yes it shuold be. I calculate pad_left it since indent is a parameter, and so it could change if one change indent parameter. But even indent is a magic number for embl: shuld I assign all those value a magic number?